src/lgr_advanced/lgr_editor/tests/test_validating_repertoire.py

```diff
@@ -0,0 +1,27 @@
+from lgr_advanced.lgr_editor.forms import ImportLGRForm
+from tests.lgr_webclient_test_base import LgrWebClientTestBase
+
+
+class TestValidatingRepertoire(LgrWebClientTestBase):
+    test_label = 'ب'  # arab character valid since the beginning (rz lgr version 1)
+    dropdown_label = 'RZ-LGR 4'
+    dropdown_labels = ['RZ-LGR 4']
+
+    def test_validating_repertoire(self):
+        dropdown = ImportLGRForm()
+        values = [v[1] for v in dropdown.fields['validating_repertoire'].choices]
+        self.assertListEqual(values, self.dropdown_labels)
+
+    def test_validating_repertoire_full(self):
+        self.login()
+
+        response = self.client.get('/a/editor/import/')
+        dropdown = response.context['form'].fields['validating_repertoire']
+        values = [v[1] for v in dropdown.choices if v[1]]
+        self.assertListEqual(values, self.dropdown_labels)
+        with open('src/lgr_web/resources/repertoires/lgr-4-common-05nov20-en.xml', 'rb') as fp:
```

Comment:
those resources should not exist anymore as the point of this update was to stop using them.
there is another reference for this in src/lgr_web/resources/lgr but this one should also be removed.
However, we need one for the initial data migration but using the one from the lgr folder is no more relevant.
What I would to is copy the lgr4-common insrc/lgr_web/resources/idn_ref/root-zone/, then update the initial data migration path and update this path as well.

Reply:
I didn't know we were about to delete them totally. Can I just but this file in a test resource? I just need any file to fill the form with valid input. Don't really care which file it is